test/views/MentionsAndSentiments/test.index.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import Accordion from '../../../views/Accordion/index.jsx';
+import MentionsAndSentiments from '../../../views/MentionsAndSentiments/index.jsx';
```

Comment:
Something for a future PR, I think you'll want to either alias directories, resolve imports starting with the app entry point, or maybe include tests along side of the actual component. This pattern is going to be brutal in the future.